File: Nível 3/1013 - O Maior/main.lua
valor = io.read()
x1,x2,x3 = valor:match("(%S+)%s+(%S+)%s+(%S+)")
x1,x2,x3 = tonumber(x1),tonumber(x2),tonumber(x3)

formula = (x1 + x2 + math.abs(x1 - x2)) / 2
resultado = (formula + x3 + math.abs(formula - x3)) / 2

print(string.format("%d eh o maior", resultado))

File: Nível 3/1013 - O Maior/main.py
valor = input("a: ")

valorA,valorB,valorC = valor.split()

numero1 = int(valorA)
numero2 = int(valorB)
numero3 = int(valorC)

formula = (numero1 + numero2 + abs(numero1 - numero2)) / 2
resultado = (formula + numero3 + abs(formula - numero3)) / 2

print("%d eh o maior" %resultado)